tools/issue-labeler/labels.go

```diff
@@ -2,30 +2,28 @@ package main
 
 import (
 	"regexp"
-	"sort"
 	"strings"
 
 	"github.com/golang/glog"
 )
 
-func labels(issueBody string, enrolledTeams map[string][]string) string {
+func labels(issueBody string, enrolledTeams map[string][]string, quoted bool) []string {
```

Comment:
Nit: I wonder if this should be named something like `serviceLabels` now that it doesn't add all of the labels (ie. `forward/review`)

Another nit: As an alternative to the `quoted` parameter, which makes the quoting a concern of this function, you could have it always return the unquoted result, and then add the quotes as part of the `fmt.Println` call in main. It's potentially a bit cleaner, and I don't think performance will be an issue here.

Reply:
Done